src/features/MentorPage/MentorSearch/MentorSearch.test.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * @jest-environment jsdom
+ */
+import { render } from '@testing-library/react';
+import MentorSearch from './MentorSearch';
+import { BrowserRouter } from 'react-router-dom';
+
+/**
+ * When functionality is added, test for search results should be added
+ */
```

Comment:
These kinds of comments will eventually pollute the codebase. There will be comments like "Fix this when bla bla"
I'd rather these things live in ticketing systems :p